core/src/test/java/org/apache/brooklyn/enricher/stock/EnrichersTest.java

```diff
@@ -201,17 +201,16 @@ public void testTransformingRespectsUnchangedButWillRepublish() {
 
         entity.sensors().set(STR1, "myval");
         Asserts.eventually(Suppliers.ofInstance(record), CollectionFunctionals.sizeEquals(1));
```

Comment:
As an aside, I noticed recently that `Asserts.eventually` has only a one second timeout, rather than the 30 second timeout of `Asserts.succeedsEventually` or `EntityAsserts.assertAttributeEqualsEventually`. I really don't like one second timeouts. They can fail in apache's jenkins if the VM/container is starved for a short length of time (admittedly I haven't seen that in a while, but I put that down to having changed lots of timeouts to use the `Asserts.DEFAULT_LONG_TIMEOUT`).

Anyway, that's not related to this PR.